cmd/bootstrap/main.go

```diff
@@ -73,19 +73,27 @@ func main() {
 	if err != nil {
 		logger.Fatal(err.Error())
 	}
+	var exitCode int
+	defer func() {
+		os.Exit(exitCode)
+	}()
 	defer db.Close()
 
 	// Create new redis client for bootstrap event store
 	esClient, err := redisClient.Setup(envPrefixES)
 	if err != nil {
-		logger.Fatal(fmt.Sprintf("failed to setup %s bootstrap event store redis client : %s", svcName, err))
+		logger.Error(fmt.Sprintf("failed to setup %s bootstrap event store redis client : %s", svcName, err))
```

Comment:
By improving the logger function can achieve the same with less line 

example:

```go
logger.Fatal(fmt.Sprintf("failed to setup %s bootstrap event store redis client : %s", svcName, err), 1)
```
in `logger/logger.go`
```go
func (l logger) Fatal(msg string, ec ...int) {
	_ = l.kitLogger.Log("fatal", msg)
	if len(ec) > 1 {
		os.Exit(ec[0])
		return
	}
	os.Exit(1)
	return
}

```

Reply:
certainly but I was looking at leaving the logger.Fatal as analogous to the standard library log.Fatal

Reply:
Let's keep it like @SammyOina suggested since the whole point of the PR is to avoid Fatal calls; i.e. to have a single Exit call.